### DISCUSSION TITLE
pretty sure somebody copied you

### DISCUSSION THREAD

**Discussion by PsychoManiac189** [RoguedBear/BetterDiscordPlugin-AutoIdleOnAFK#16] [General]
https://github.com/RoguedBear/BetterDiscordPlugin-AutoIdleOnAFK/blob/5763a1dc08692d9fae8147b0fe7c6370dba14d51/release/AutoIdleOnAFK.plugin.js

**Comment by RoguedBear** [comment]
can you elaborate more please?
